### PR TITLE
chore: combine storybook related npm scripts

### PR DESCRIPTION
**Pull request by MkMan**
* ~Added building storybook to the deploy to GH pages script seeing that it's a prerequisite for it~ REVERTED
* Deleted the `storybook` script because npm `start` builds and serves 
* Updated docs to match
* Updated Storybook and its plugins to the latest version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,6 @@
     "release": "npm run build:prod",
     "sanity-check": "npm run format && npm run lint && npm test",
     "start": "npm run storybook",
-    "storybook": "start-storybook -p 6006  -s ./public",
```
Comment (beau-charman) [inline package.json:26]: If you remove L26, L25 will fail

File: documentation/welcome.stories.mdx [hunk 2]
```diff
@@ -74,10 +74,10 @@ npm run test -- -u
 
 ### UI and Design Testing
 
-Story Book is used to Visually test and showcase components. Run the following to build the storybook files and serve them on `http://localhost:6006/?path=/story/*` respectively:
+Story Book is used to Visually test and showcase components. Run the following to build the storybook files and serve them on `http://localhost:6006/?path=/story/*`:
 
 ```
-npm run build-storybook && npm run storybook
+npm start
```
Comment (beau-charman) [inline documentation/welcome.stories.mdx:80]: Thank you for updating these

File: package.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
     "build:dev": "NODE_ENV=development rollup -c",
     "build:prod": "NODE_ENV=production rollup -c",
     "build:watch": "NODE_ENV=production rollup -c -w",
-    "deploy-storybook": "gh-pages -d storybook-static",
+    "deploy-storybook": "build-storybook -s ./public && gh-pages -d storybook-static",
```
Comment (beau-charman) [inline package.json:17]: I'd like to keep the deploy task seperate from the build task, could we at least have a `build-deploy-storybook` task

**Review by beau-charman — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (MkMan) [reply package.json:26]: Good pick up, can't believe I missed that 🤦‍♂️ 
Should I still remove L26 and just make the start script `"start-storybook -p 6006  -s ./public"`

Comment (beau-charman) [reply package.json:26]: Haha it's okay!

File: package.json (hunk 3 above)
Comment (MkMan) [reply package.json]: Fair enough 👍  I have reverted this change

File: package.json (hunk 1 above)
Comment (MkMan) [reply package.json:26]: I updated the "start" script to do what the "storybook" script used to do.

**Pull request merged**